maro/simulator/scenarios/supply_chain/sku_dynamics_sampler.py

```diff
@@ -0,0 +1,294 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from __future__ import annotations
+
+import os.path
+import typing
+import warnings
+from abc import ABCMeta, abstractmethod
+from collections import defaultdict, namedtuple
+from csv import DictReader
+from datetime import datetime
+from typing import Dict, Optional, Union
+
+from dateutil.parser import parse
+from tqdm import tqdm
+
+from maro.data_lib.supply_chain import (
+    DATE_INDEX_COLUMN_NAME, get_date_index, get_preprocessed_file_path, preprocess_file
+)
+
+from .units.seller import SellerDemandInterface
```

Comment:
`XXXInterface` is a Java-style naming. According to our previous experiences, in Python, the better naming is `XXXMixin`